src/main/java/jenkins/plugins/office365connector/util/FormUtils.java

```diff
@@ -0,0 +1,36 @@
+package jenkins.plugins.office365connector.util;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import hudson.util.FormValidation;
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Collects method for forms.
+ *
+ * @author Joseph Petersen (casz@github)
+ */
+public class FormUtils {
+
+    /**
+     * Validates data is in expected format either URL or variable reference.
+     * @param value user input to validate
+     * @return OK or Error on pending user input
+     */
+    public static FormValidation validateUrl(String value) {
+        if (StringUtils.isBlank(value)) {
+            return FormValidation.error("URL or variable reference must be provided");
+        }
+        if (value.startsWith("$") && value.length() > 1) {
+            return FormValidation.ok();
+        }
+
+        try {
+            new URL(value);
+        } catch (MalformedURLException e) {
+            return FormValidation.error(e.getMessage());
+        }
+        return FormValidation.ok();
```

Comment:
can you provide unit test for this method?

Reply:
With minimum="0" removed: 
```
properties([office365Webhooks([[macros: [[]], url: '']])])
```

![image](https://user-images.githubusercontent.com/1661688/31359529-9f850bf8-ad4a-11e7-8a10-3de555b88a7e.png)


With zero:
```
properties([office365Webhooks([[url: '']])])
```
![image](https://user-images.githubusercontent.com/1661688/31359434-47913660-ad4a-11e7-8b32-e9c98a1382d8.png)

Reply:
Done, but while writing I figured out a case we don't cover 👍